tests/generators/OperationGenerator.ts

```diff
@@ -462,7 +454,7 @@ export default class OperationGenerator {
    */
   public static async createUpdateOperationRequestForAddingAKey (
     didUniqueSuffix: string,
-    updateRevealValue: string,
+    updateKey: JwkEs256k,
```

Comment:
Consider not requiring to pass the public key, since private key is passed already you can use `getEs256kPublicKey()` internally.. The goal being trying to make the method as easy to use as possible.

Reply:
We need it for some tests that test when the key is invalid

Reply:
Agree based on the principal that it is fine for test libraries to be more flexible and less usable.